cluster-controller/src/main/java/io/strimzi/controller/cluster/resources/KafkaCluster.java

```diff
@@ -34,7 +34,7 @@
     protected static final int REPLICATION_PORT = 9091;
     protected static final String REPLICATION_PORT_NAME = "replication";
 
-    private static final String NAME_SUFFIX = "-kafka";
+    protected static final String NAME_SUFFIX = "-kafka";
```

Comment:
Why did you need to make this `public`? We should try to use the methods which encapsulate the name generation.

Reply:
You are right. Fixed.